balto-ui/src/components/DogSearchPage/DogSearchPage.jsx

```diff
@@ -33,10 +33,14 @@ export default function DogSearchPage() {
     kidFriendly: selectedGoodWith.includes("Kids"),
     strangerFriendly: selectedGoodWith.includes("Strangers"),
     dogFriendly: selectedGoodWith.includes("Other dogs"),
+    catFriendly: selectedGoodWith.includes("Cats"),
+    noviceFriendly: selectedGoodWith.includes("Beginner Owners"),
     distance: selectedDistance || null,
     shelterIds: selectedShelters
   }
 
+  console.log(filters)
```

Comment:
https://www.overleaf.com/read/cjgcxvypwrhk

Reply:
Removed in most recent commit